spec/services/moment_keywords_spec.rb

```diff
@@ -0,0 +1,45 @@
+
+describe MomentKeywords do
+    let(:user) { create(:user) }
+    let(:categories) {[FactoryBot.build(:category,
+                                        name: "category name",
+                                        description: "category description."
+    )]}
```

Comment:
For multiline blocks, the most accepted style is to use `do...end`

```suggestion
    let(:categories) do
      FactoryBot.build([
        :category,
        name: "category name",
        description: "category description."
      )]
    end
```